Makefile

```diff
@@ -149,14 +149,15 @@ TIMEOUT  = 10m
 TESTPKGS = $(shell env GO111MODULE=on $(GO) list -f \
             '{{ if or .TestGoFiles .XTestGoFiles }}{{ .ImportPath }}{{ end }}' \
             $(PKGS))
-TEST_TARGETS := test-default test-bench test-short test-verbose test-race
+TEST_TARGETS := test-default test-bench test-short test-verbose test-race check-ci
 test-bench:   ARGS=-run=__absolutelynothing__ -bench=.
 test-short:   ARGS=-short
 test-verbose: ARGS=-v
 test-race:    ARGS=-race
+check-ci: 	  ENV_VARS=INTEGRATION_TESTS=true
```

Comment:
Personally I would have called this `check-integration` or something similar, as I'm more interested in what the target does than the reason or use case for running it. Since I'm going to use this locally as well as in CI, it also doesn't reflect the actual usage. But check-ci is nice and short though. And not a big issue, if you see it from another perspective just go ahead an merge.